public/app.json

```diff
@@ -1,3 +1,3 @@
 {
-  "version": "0.0.0"
+  "version": "0.16.0"
```

Comment:
this is automatically updated by a GitHub Action - changing this will break the GitHub Action because it looks for `0.0.0`

Reply:
Should i push another commit to revert it ?

Reply:
Done !!!